fundingmanager.go

```diff
@@ -483,11 +496,21 @@ func (f *fundingManager) Start() error {
 
 		case fundingLockedSent:
 			// fundingLocked was sent to peer, but the channel
+			// was not added to the router graph and the channel
 			// announcement was not sent.
 			f.wg.Add(1)
 			go func() {
 				defer f.wg.Done()
 
+				// Retrieve channel announcement preference
```

Comment:
This block of code is starting to sprawl a bit. It should be extract into a distinct method for readability purposes. This should likely be done in a distinct commit _before_ the new logic is added, as this will make the change easier to review, and will also make the change (move only) explicit in the git history.